app/main.py

```diff
@@ -1,8 +1,17 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    persons_list = [Person(person["name"], person["age"])for person in people]
+    for person in people:
+        iterable_person = Person.people[person["name"]]
+        if "wife" in person and person["wife"] is not None:
+            iterable_person.wife = Person.people[person["wife"]]
```

Comment:
There is a potential issue here. Before setting `iterable_person.wife`, you should check if `person["wife"]` exists in `Person.people` to avoid a `KeyError`. Consider using `Person.people.get(person["wife"])` and handle the case where it returns `None`.